src/trading/types.ts

```diff
@@ -0,0 +1,161 @@
+import type { AppDataParams, latest, LatestAppDataDocVersion } from '@cowprotocol/app-data'
+import {
+  AppData,
+  AppDataHash,
+  OrderKind,
+  OrderParameters,
+  OrderQuoteRequest,
+  OrderQuoteResponse,
+  QuoteAmountsAndCosts,
+  TokenAmount,
+} from '../order-book'
+import type { Signer } from '@ethersproject/abstract-signer'
+import type { CowEnv, SupportedChainId } from '../common'
+import type { ExternalProvider } from '@ethersproject/providers'
+import type { UnsignedOrder } from '../order-signing'
+
+export type PrivateKey = string // 64 characters
+export type AccountAddress = `0x${string}` // 42 characters
```

Comment:
Since address has the `0x${string}` patterns, would it make sense to have it for PrivateKey as well?

Reply:
Ethers Signer actually allows using privateKey without `0x` prefix